packages/core/admin/admin/src/pages/UseCasePage.tsx

```diff
@@ -69,19 +69,22 @@ const TypographyCenter = styled(Typography)`
   text-align: center;
 `;
 
-const UseCasePage = () => {
+export const UseCasePage = () => {
   const toggleNotification = useNotification();
   const { push, location } = useHistory();
   const { formatMessage } = useIntl();
-  const [role, setRole] = useState();
-  const [otherRole, setOtherRole] = useState('');
+  const [role, setRole] = React.useState<string | number | null>(null);
+  const [otherRole, setOtherRole] = React.useState('');
   const { post } = useFetchClient();
 
-  const { firstname, email } = auth.getUserInfo();
+  const { firstname, email } = auth.get('userInfo');
```

Comment:
@joshuaellis I've changed this because `getUserInfo` is deprecated.

What is the recommended way to say this returns `UserInfo` instead of string or null?

Reply:
I removed `string` in #18473 because it felt wrong – we should be using the interface map, to check if it's null I suppose you could `?? {}` that way if for whatever reason we can't get it from local/session storage it doesn't crash? 🤷🏼‍♀️